theano/sandbox/scan_module/scan_op.py

```diff
@@ -342,10 +342,10 @@ def rval(p=p, i=node_input_storage, o=node_output_storage, n=node):
         return rval
 
     def grad(self, args, g_outs):
-        pass
+        raise NotImplementedError()
 
     def R_op(self, inputs, eval_points):
-        pass
+        raise NotImplementedError()
```

Comment:
R_op in PureOp class in theano/sandbox/scan_module/scan_op.py does raise NotImplementedError(), but with a message. If this is the signal that is missing, I can add that here. What I meant was that there are many gradient methods in theano that just call NotImplementedError() to signal a problem. Now, if it is not the right behaviour, that should get changed. What I meant with "default implementation" was avoiding returning [None] in any case. I can remove these two functions if you think this is a better approach, but if this approach (raising NotImplementedError) is wrong, it should get changed inside the whole theano and not just here.